platformtour/static/css/platformtour_studio.css

```diff
@@ -0,0 +1,38 @@
+#platformtour_studio_step_choices {
```

Comment:
Is there a common parent selector/wrapper that we can use to prefix all of these selectors?
Similar to what we usually do on the LMS..

Reply:
`.xblock_platformtour_studio`?